src/Core/tests/DeviceTests/Handlers/Entry/EntryHandlerTests.Android.cs

```diff
@@ -15,6 +15,46 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public partial class EntryHandlerTests
 	{
+		[Theory(DisplayName = "Validates Keyboard updates correctly using IsPassword")]
+		[InlineData(nameof(Keyboard.Text), false)]
+		[InlineData(nameof(Keyboard.Numeric), true)]
+		public async Task ValidateKeyboardUpdatesCorrectly(string keyboardName, bool expected)
```

Comment:
Not sure if my suggestion here is the best, but it would be great to reword the test name to more clearly reflect what the test does

```suggestion
		public async Task ValidateNumericKeyboardUpdatesCorrectlyWithIsPassword(string keyboardName, bool expected)
```